source/FreeRTOS_UDP_IPv6.c

```diff
@@ -205,7 +218,7 @@ void vProcessGeneratedUDPPacket_IPv6( NetworkBufferDescriptor_t * const pxNetwor
     NetworkInterface_t * pxInterface = NULL;
     EthernetHeader_t * pxEthernetHeader = NULL;
     BaseType_t xLostBuffer = pdFALSE;
-    NetworkEndPoint_t * pxEndPoint = NULL;
+    NetworkEndPoint_t * pxEndPoint = pxNetworkBuffer->pxEndPoint;
```

Comment:
The pxEndPoint is getting updated by eNDGetCacheEntry, What is need of initilializing here?